portal/flatpak-portal.c

```diff
@@ -848,6 +851,21 @@ handle_spawn (PortalFlatpak         *object,
   instance_path = g_key_file_get_string (app_info,
                                          FLATPAK_METADATA_GROUP_INSTANCE,
                                          FLATPAK_METADATA_KEY_INSTANCE_PATH, NULL);
+
+  /* TODO: Should this be conditional on a flag, so that e.g. Steam can
+   * share /tmp with some sandboxes but not with others? */
```

Comment:
Maybe we want `FLATPAK_SPAWN_FLAGS_SHARE_DEV_SHM`, `_TMP`? Or maybe the default should be for apps that have the right flag in their metadata to share their /dev/shm and /tmp with subsandboxes, but there should be a flag to *unshare* it?

Also, how should this interact with `FLATPAK_SPAWN_FLAGS_SANDBOX`? Should those more restricted subsandboxes have their own `/dev/shm` and `/tmp` by default?

(I don't really mind how this works for Steam, because Steam is not currently aiming to put up any privilege boundary between Steam and games - although it'd be nice if we could, one day.)

Reply:
Now that it's an `--allow` feature, `FLATPAK_SPAWN_FLAGS_SANDBOX` automatically disables it, which is nice.